README.md

```diff
@@ -34,6 +34,96 @@ The `DEBUG` environment variable can be used to activate logs, ex:
 DEBUG="onyx*" onyx-server
 ```
 
+### Setting up an Onyx Server on AWS
+
+You can use a pre-built [AMI](https://en.wikipedia.org/wiki/Amazon_Machine_Image)
+to conveniently set up an onyx-server on AWS.
+
+To do it, make sure you have an AWS account and your AWS CLI is configured to 
+use the `eu-west-1` (Ireland) region as default. We're going to assume you have a
+[VPC](https://eu-west-1.console.aws.amazon.com/vpc/home?region=eu-west-1#)
+configured (created and connected to an Internet Gateway) in this region.
```

Comment:
Aside from a VPC and an Internet Gateway, Route Table has to be configured, with a route set like this:

| Destination        | Target                                                |  Status   | Propagated |
| ---                      | ----                                                      | ---          | ---                |
| 10.0.0.0/16        | local                                                   |  Active   | No               |
| 0.0.0.0/0            | \<INTERNET GATEWAY ID HERE> |  Active   | No               |

Reply:
That's kind of what I meant in `created and connected to an Internet Gateway`. I'll add a link to the Route Tables there to make it more explicit what needs to happen.